cron_scripts/lib/ZCS/API.pm

```diff
@@ -0,0 +1,343 @@
+package ZCS::API;
```

Comment:
Rename the module to some other custom name?  Maybe ZCS::API::Local or ZCS::API::Custom or ...